builtin/game/register.lua

```diff
@@ -242,6 +265,19 @@ function core.register_alias(name, convert_to)
 	end
 end
 
+function core.register_alias_force(name, convert_to)
+	if forbidden_item_names[name] then
+		error("Unable to register alias: Name is forbidden: " .. name)
+	end
+	if core.registered_items[name] ~= nil then
+		core.unregister_item(name)
+		core.log("info", "Erased item " .. name .. " because registering a force alias with the same name.")
```

Comment:
The sentence is completely ungrammatical anyway.  Change it to "Removed item " ..name.." while attempting to force add an alias" or similar.